packages/google_identity_services_web/README.md

```diff
@@ -0,0 +1,70 @@
+# google_identity_services_web
+
+A js-interop layer for Google Identity's Sign In With Google SDK.
+
+See the original JS SDK reference:
+
+* [Sign In With Google](https://developers.google.com/identity/gsi/web)
+
+## Usage
+
+This package is the Dart js-interop layer of the new **Sign In With Google**
+SDK. Here's the API references for both of the sub-libraries:
+
+* `id.dart`: [Sign In With Google JavaScript API reference](https://developers.google.com/identity/gsi/web/reference/js-reference)
+* `oauth2.dart`: [Google 3P Authorization JavaScript Library for websites - API reference](https://developers.google.com/identity/oauth2/web/reference/js-reference)
+* `loader.dart`: An (optional) loader mechanism that installs the library and
+resolves a `Future<void>` when it's ready.
+
+### Loading the SDK
+
+There's two ways to load the JS SDK in your app. The most performant way is to
+modify your `web/index.html` file to insert the script tag [as recommended](https://developers.google.com/identity/gsi/web/guides/client-library). Place the `script` tag in the
+`<head>` of your site, next to the script tag that loads `flutter.js`, so both
+can be downloaded in parallel:
+
+```html
+<!-- Your index.html -->
+<head>
+    ...
+    <script src="https://accounts.google.com/gsi/client" async defer></script>
+    <!-- ^^^^^^^-- add the new script there -->
+    <script src="flutter.js" defer></script>
+    ...
+</head>
+```
+
+An alternative way, that downloads the SDK on demand, is to use the
+**`loadWebSdk`** function provided by the library. A simple location to embed
+this in a Flutter Web only app can be the `main.dart`:
+
+```dart
+...
+import 'package:google_identity_services_web/loader.dart' as gis;
+
+void main() async {
+  await gis.loadWebSdk(); // Load the GIS SDK
+  ...
+  runApp(const MyApp());
+}
+```
+
+(Note that the above won't compile for mobile apps, so if you're developing a
+cross-platform app, you'll probably need to hide the call to `loadWebSdk`
+behind a [conditional import/export](https://dart.dev/guides/libraries/create-library-packages#conditionally-importing-and-exporting-library-files).)
+
+## Browser compatibility
+
+The new SDK is introducing new concepts that are on track of standardization to
+most browsers, and older browsers might not be compatible with older browsers.
```

Comment:
"and might not be compatible with older browsers" (removes duplicate "older browsers").